iocore/cache/CacheRead.cc

```diff
@@ -254,7 +254,6 @@ CacheVC::openReadChooseWriter(int /* event ATS_UNUSED */, Event * /* e ATS_UNUSE
           continue;
         }
       }
-      ink_assert(w->alternate.valid());
```

Comment:
This assertion is triggered with missing lock. It is hardly for us to debug this lock issue if we remove this.